### PR TITLE
Annualized costs

### PR DESCRIPTION
**Pull request by NienkeWagenaar**
## Goal
Add amortization rate for CAPEX investments, using following formula: http://www.energycommunity.org/webhelppro/Expressions/AnnualizedCost.htm
 
### Changes to Excel: 
 
•	Addition of tab Economics
 
 
### Changes to code: 
 
• Run_strategic_model.py
	Update parameter_list to included Economics tab
• Strategic_produced_water_optimization.py
	o Def _preprocess_data
              •	Create AnnualizationRate
              •	Calculated based on new inputs: discount rate and CAPEX_lifetime
        o Added p_alpha_AnnualizationRate as parameter
              •	Default 1
              •	Initialized with AnnualizationRate
         o	CostObjectiveFunctionRule
                •	Multiplied CAPEX components in objective function with annualization rate

 
### Documentation
 
•	Added Alpha_annualization rate
•	Updated objective function

### Assumptions
•	Discount rate and life expectancy is the same for all types of CAPEX expenses.  
•	In the reporting, we want to see the full CAPEX expense, not multiplied with the annualization rate. Therefore, I've only added the annualization rate in the objective function, not in the variables itself.

### CODE REVIEW

File: pareto/strategic_water_management/strategic_produced_water_optimization.py [hunk 1]
```diff
@@ -1094,10 +1120,13 @@ def CostObjectiveFunctionRule(model):
...
-                + model.v_C_StorageCapEx
-                + +model.v_C_TreatmentCapEx
-                + model.v_C_PipelineCapEx
+                + model.p_alpha_AnnualizationRate
+                * (
+                    model.v_C_DisposalCapEx
+                    + model.v_C_StorageCapEx
+                    + model.v_C_TreatmentCapEx
+                    + model.v_C_PipelineCapEx
+                )
```
Comment (MAZamarripa) [inline pareto/strategic_water_management/strategic_produced_water_optimization.py:1129]: missing annualization rate for reuse objective function

**Review by MAZamarripa — CHANGES_REQUESTED** [review]
update branch after merging PR #81

**Review by melody-shellman — APPROVED** [review]
Looks good. Regarding the compatibility with the pipeline config options, I see no issues.

File: pareto/strategic_water_management/strategic_produced_water_optimization.py (hunk 1 above)
Comment (NienkeWagenaar) [reply pareto/strategic_water_management/strategic_produced_water_optimization.py:1129]: Thanks! 

Related to PR #81: Melody merged in applicable changes in commit 4ea6a5a.

**Comment by NienkeWagenaar** [comment]
@MAZamarripa, Related to PR #81: Melody merged in applicable changes in commit 4ea6a5a.

If no additional comments, I think this PR is ready to be merged. 
Regarding the compatibility with the pipeline config options, the only issue I see is that the Excel file should be merged.

**Review by MAZamarripa — APPROVED** [review]
lgtm

**Pull request merged**